virtio-devices/src/balloon.rs

```diff
@@ -233,7 +340,12 @@ impl BalloonEpollHandler {
             let mut descs_len = 0;
             while let Some(desc) = desc_chain.next() {
                 descs_len += desc.len();
-                Self::release_memory_range(desc_chain.memory(), desc.addr(), desc.len() as usize)?;
+                Self::release_memory_range(
+                    &mut None,
+                    desc_chain.memory(),
+                    desc.addr(),
+                    desc.len() as usize,
+                )?;
             }
```

Comment:
Keep the original `release_memory_range` as it is to avoid any changes here.

Reply:
Yes.